scripts/check-fix-standards.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
Pourquoi mettre cette ligne dans tous les scripts ?

Reply:
c'est pas obligatoire mais mieux visiblement
https://superuser.com/questions/1113014/what-would-set-e-and-set-x-commands-do-in-the-context-of-a-shell-script

https://www.davidpashley.com/articles/writing-robust-shell-scripts/#id2382181

Reply:
Ok.